DFC.Digital/DFC.Digital.Web.Sitefinity.CmsExtensions.UnitTests/Controllers/ApprenticeshipVacancyReportControllerTests.cs

```diff
@@ -73,13 +76,17 @@ public void IndexTest(int numberRecords, int numberOfApprenticeship)
 
         }
 
-        public void IndexRedirectTest(int numberRecords, int numberOfApprenticeship)
+        [Fact]
+        public void IndexRedirectTest()
         {
             // Setup
-            A.CallTo(() => fakeReportRepository.GetJobProfileApprenticeshipVacancyReport()).Returns(GetDummyReportData(numberRecords, numberOfApprenticeship));
-            A.CallTo(() => fakeWebAppContext.RequestQueryString).Returns(null);
+            A.CallTo(() => fakeReportRepository.GetJobProfileApprenticeshipVacancyReport()).Returns(GetDummyReportData(1, 2));
+           // A.CallTo(() => fakeWebAppContext.RequestQueryString).Returns(null);
```

Comment:
Why can't we fake call to get property?
```suggestion
           query.Add("NOTctx", "something");
           A.CallTo(() => fakeWebAppContext.RequestQueryString).Returns(query);
```